src/psqlgraph/poly.py

```diff
@@ -0,0 +1,52 @@
+"""A module for poly edges/nodes which generate an entity based on a label."""
+
+from __future__ import annotations
+
+from collections.abc import Mapping
+from typing import Any
+
+from psqlgraph import edge, node
+
+
+def PolyEdge(
+    *,
+    label: str,
+    src_id: str | None = None,
+    dst_id: str | None = None,
+    acl: list[str] | None = None,
+    system_annotations: Mapping[str, Any] | None = None,
+    properties: Mapping[str, Any] | None = None,
+):
+    edge_cls = edge.Edge.get_subclass(label)
+
+    if not edge_cls:
```

Comment:
I think the null check for label should be kept since its still possible to do `label=None`

Reply:
Not sure it makes sense to have a specific check for a None. Worse, edge and node were already doing the check differently (one assert the other if/raise.) So this standardizes it a bit. Ultimately tho, None will behave like any other bad label and I don't think we should for people completely disregarding their linter.